boards/xtensa/intel_adsp_cavs15/tools/flash.sh

```diff
@@ -5,14 +5,16 @@
 BUILD=$1
 FIRMWARE=${BUILD}/zephyr/zephyr.ri
 FLASHER=${ZEPHYR_BASE}/boards/xtensa/intel_adsp_cavs15/tools/fw_loader.py
+CONFIGURATION=${ZEPHYR_BASE}/../modules/audio/sof/zephyr/ext/rimage/config
+RIMAGE=${ZEPHYR_BASE}/../modules/audio/sof/zephyr/ext/rimage/build/rimage
```

Comment:
is rimage built there automatically? That seems strange - west only builds everything in a separate directory, as far as I understand?

Reply:
rimage build is not automatically, need build and install by ourselves, then we can use twister to run kernel tests.

Reply:
@KangJianX but then CI cannot use it, can it?

Reply:
What is your means that CI cannot use it?

Reply:
@KangJianX I mean would you program CI to build rimage additionally?

Reply:
No, I won't build it additionally